docker/bin/e2e/mysql/e2e-env-reset.sh

```diff
@@ -19,15 +19,15 @@ mysql -h $MYSQL_HOST -u $MYSQL_USER -p$MYSQL_PASSWORD -e "DROP DATABASE IF EXIST
 # Tracker
 
 # Delete tracker database
-rm -f ./storage/database/torrust_tracker_e2e_testing.db
+rm -f ./storage/tracker/lib/database/torrust_tracker_e2e_testing.db
 
 # Generate storage directory if it does not exist
-mkdir -p "./storage/database"
+mkdir -p "./storage/tracker/lib/database"
 
 # Generate the sqlite database for the tracker if it does not exist
-if ! [ -f "./storage/database/torrust_tracker_e2e_testing.db" ]; then
-    touch ./storage/database/torrust_tracker_e2e_testing.db
-    echo ";" | sqlite3 ./storage/database/torrust_tracker_e2e_testing.db
+if ! [ -f "./storage/tracker/lib/database/torrust_tracker_e2e_testing.db" ]; then
+    touch ./storage/tracker/lib/database/torrust_tracker_e2e_testing.db
+    echo ";" | sqlite3 ./storage/tracker/lib/database/torrust_tracker_e2e_testing.db
```

Comment:
I recommend that you create the empty database with this command:

https://github.com/torrust/torrust-tracker/blob/698890a0b1dcf696013af640a131a7c146d295ed/Containerfile#L20-L21

Reply:
Yes, thank you! I saw the other day you were using it. I will change it.